lib/renderer/api/remote.js

```diff
@@ -43,17 +41,17 @@ function wrapArgs (args, visited = new Set()) {
       }
       visited.delete(value)
```

Comment:
it's still a mystery to me why we have two separate serialization algorithms

Reply:
do you mean main -> renderer and renderer -> main?

Reply:
yes